src/viam/examples/modules/tflite/main.cpp

```diff
@@ -73,7 +73,7 @@ class MLModelServiceTFLite : public vsdk::MLModelService {
         // drain.
     }
 
-    void stop(const vsdk::AttributeMap& extra) noexcept final {
+    void stop(const vsdk::AttributeMap& extra) noexcept final override {
```

Comment:
Because there is nothing actually *wrong* with doing it. It's just redundant. I think clang-tidy does have a rule for it it we want: https://clang.llvm.org/extra/clang-tidy/checks/modernize/use-override.html

Reply:
No strong opinion; seems like if you're at all familiar with what `override` and `final` do, you would not use both accidentally (I just didn't know what `final` was tbh).